### DISCUSSION TITLE
In game lags

### DISCUSSION THREAD

**Discussion by TrueCrashB** [darthcloud/BlueRetro#616] [General]
After installing blue retro on my Nintendo Gamecube with picoboot i have ingame lags. No input lags. The games are stuttering sometimes. I used the original cabled GC Controller. Any idea?

**Comment by darthcloud** [comment]
Update BlueRetro to v1.8.3.

Previous version are holding system reset constantly.
Alternatively you can press reset button once to unstuck it with the old version.
